### PR TITLE
Idv 223

### PR DESCRIPTION
**Pull request by janireu**

### CODE REVIEW

**Review by AndyRigley — COMMENTED** [review]
Tested locally and works fine.
Two questions though as discussed:
1. Do we care or need to trap a verified student with NO Institution? (this can happen if you man. verify in admin)
2. Do we want to add city to the message (**We can see you are currenty studying in Abbey College (Manchester).** ) to reduce ambiguity where >1 college,  and bring more in line with WAYF displays.

File: src/vue/src/views/CompleteSignup.vue [hunk 1]
```diff
@@ -14,6 +14,32 @@
...
+                <button
+                    class="power_button effect_1 tell-us-more"
+                    v-if="!showInstitutionInfo"
+                >
+                    <span class="button_value" @click="retrieveInstitutionInfo"
+                        >Tell us more about your institution!</span
+                    >
+                </button>
+                <p v-if="showInstitutionInfo">
+                    We can see you are currenty studying in
```
Comment (AndyRigley) [inline src/vue/src/views/CompleteSignup.vue:31]: typo - should be currently

**Comment by janireu** [comment]
I've just double checked the Api call to get the institution data and it does not contain any address information, so we cannot display it there unless we enhance it.

Regarding the 404, I've added a friendly message for it

**Comment by AndyRigley** [comment]
> I've just double checked the Api call to get the institution data and it does not contain any address information, so we cannot display it there unless we enhance it.
> 
> Regarding the 404, I've added a friendly message for it

Correct, we have to add it to the json payload and return it...

**Pull request merged**